src/Test/Integration/Model/Service/ExportTest.php

```diff
@@ -375,6 +377,249 @@ public function testCheckCartCoupons($expected, $cart)
+     * @throws NoSuchEntityException
+     * @throws Zend_Json_Exception
+     */
+    public function testCheckCartShippingMethodsNetWithoutTaxNoCrossBoarder()
+    {
+        $expectedAmounts = [
+            'amount' => 15,
+            'amount_with_tax' => 15,
+            'tax_percent' => 0,
+        ];
+
+        $this->runCheckCartShippingTest($expectedAmounts);
+    }
+
+    /**
+     * @magentoConfigFixture current_store tax/calculation/shipping_includes_tax 0
+     * @magentoConfigFixture current_store tax/calculation/cross_border_trade_enabled 1
+     *
+     * @throws CouldNotSaveException
+     * @throws NoSuchEntityException
+     * @throws Zend_Json_Exception
+     */
+    public function testCheckCartShippingMethodsNetWithoutTaxAndCrossBorder()
+    {
+        $expectedAmounts = [
+            'amount' => 15,
+            'amount_with_tax' => 15,
+            'tax_percent' => 0,
+        ];
+
+        $this->runCheckCartShippingTest($expectedAmounts);
+    }
+
+    /**
+     * @magentoConfigFixture current_store tax/calculation/shipping_includes_tax 1
+     * @magentoConfigFixture current_store tax/calculation/cross_border_trade_enabled 0
+     *
+     * @throws CouldNotSaveException
+     * @throws NoSuchEntityException
+     * @throws Zend_Json_Exception
+     */
+    public function testCheckCartShippingMethodsNetWithTaxNoCrossBoarder()
+    {
+        $expectedAmounts = [
+            'amount' => 15,
+            'amount_with_tax' => 15,
+            'tax_percent' => 0,
+        ];
+
+        $this->runCheckCartShippingTest($expectedAmounts);
+    }
+
+    /**
+     * @magentoConfigFixture current_store tax/calculation/shipping_includes_tax 1
+     * @magentoConfigFixture current_store tax/calculation/cross_border_trade_enabled 1
+     *
+     * @throws CouldNotSaveException
+     * @throws NoSuchEntityException
+     * @throws Zend_Json_Exception
+     */
+    public function testCheckCartShippingMethodsNetWithTaxNetAndBorderTrade()
+    {
+        $expectedAmounts = [
+            'amount' => 15,
+            'amount_with_tax' => 15,
+            'tax_percent' => 0,
+        ];
+
+        $this->runCheckCartShippingTest($expectedAmounts);
+    }
+
+    /**
+     * @param array $expectedAmounts
+     *
+     * @throws CouldNotSaveException
+     * @throws NoSuchEntityException
+     * @throws Zend_Json_Exception
+     */
+    private function runCheckCartShippingTest($expectedAmounts)
+    {
+        $cart = $this->getCartForShippingTests();
+        $internalInfo = Zend_Json_Decoder::decode($cart['cart']['items'][0]['internal_order_info']);
+        $this->stockManager->setStockWebsite($internalInfo['product_id']);
+
+        /** @var \Shopgate\Export\Model\Service\Export $class */
+        $class  = Bootstrap::getObjectManager()->create('Shopgate\Export\Model\Service\Export');
+        $return = $class->checkCart($cart);
+        $shippingMethods = array_pop($return['shipping_methods']);
```

Comment:
equal signs not aligned, also, check if there are more shipping_methods. How many are expected?